### PR TITLE
更新webpack模块热替换翻译文档

### PR DESCRIPTION
**Pull request by guyunxiang**
1. [Read and sign the CLA](https://cla.js.foundation/webpack/webpack.js.org). This needs to be done only once. PRs that haven't signed it won't be accepted.
2. Make sure your PR complies with [the writer's guide](https://webpack.js.org/writers-guide/).
3. Read through the PR diff carefully as sometimes this can reveal issues. The work will be reviewed, but this can save some effort.
4. Remove these instructions from your PR as they are for your eyes only.

### CODE REVIEW

**Comment by dear-lizhihua** [comment]
收到，最近比较忙，周末抽空审核。

File: content/guides/hot-module-replacement.md [hunk 1]
```diff
@@ -12,23 +12,24 @@ contributors:
...
-模块热替换(Hot Module Replacement 或 HMR)是 webpack 提供的最有用的功能之一。它允许在运行时更新各种模块，而无需进行完全刷新。本页面重点介绍__实现__，而[概念页面](/concepts/hot-module-replacement)提供了更多关于它的工作原理以及为什么它有用的细节。
+模块热替换(Hot Module Replacement 或 HMR)是 webpack 提供的最有用的功能之一。它允许在运行时更新各种模块，而无需进行完全刷新。本页面重点介绍 __实现__，而[概念页面](/concepts/hot-module-replacement)提供了更多关于它的工作原理以及为什么它有用的细节。
 
 W> __HMR__ 不适用于生产环境，这意味着它应当只在开发环境使用。更多详细信息，请查看[生产环境构建指南](/guides/production)。
 
 
 ## 启用 HMR
 
-启用此功能实际上相当简单。All we need to do is update our [webpack-dev-server](https://github.com/webpack/webpack-dev-server) configuration, and use webpack's built in HMR plugin. We'll also remove the entry point for `print.js` as it will now be consumed by the `index.js` module.
+启用此功能实际上相当简单。而我们要做的，就是更新 [webpack-dev-server](https://github.com/webpack/webpack-dev-server) 的设置 ，和使用 webpack 的 HMR 插件。现在我们要删除掉 `print.js` 的入口配置以及被 `index.js` 文件的索引。
```
Comment (Aladdin-ADD) [inline content/guides/hot-module-replacement.md]: >现在我们要删除掉 `print.js` 的入口配置以及被 `index.js` 文件的索引。

这句话的理解好像有点问题，这里的as是“因为”

Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: 大家多用谷歌翻译 https://translate.google.cn/m/translate，as 是因为。
所有我们需要做的是更新我们的 [webpack-dev-server](https://github.com/webpack/webpack-dev-server) 配置，并使用 webpack 内置的 HMR 插件。我们还将删除 `print.js` 入口起点，因为它现在将被 `index.js` 模块引用。

File: content/guides/hot-module-replacement.md [hunk 2]
```diff
@@ -12,23 +12,24 @@ contributors:
   - skipjack
   - sbaidon
   - gdi2290
+  - guyunxiang32
```
Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: 这里是英文维护文档人员的名字，要去掉翻译人员名字

File: content/guides/hot-module-replacement.md [hunk 3]
```diff
@@ -12,23 +12,24 @@ contributors:
...
 related:
   - title: 概念 - 模块热替换(Hot Module Replacement)
     url: /concepts/hot-module-replacement
   - title: API - 模块热替换(Hot Module Replacement)
     url: /api/hot-module-replacement
 ---
 
-T> This guide extends on code examples found in the [Development](/guides/development) guide.
+T> 本指南示例代码延用[开发](/guides/development)模块的示例代码。
```
Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: [开发](/guides/development)模块->[开发](/guides/development)指南

File: content/guides/hot-module-replacement.md [hunk 4]
```diff
@@ -12,23 +12,24 @@ contributors:
...
 
-模块热替换(Hot Module Replacement 或 HMR)是 webpack 提供的最有用的功能之一。它允许在运行时更新各种模块，而无需进行完全刷新。本页面重点介绍__实现__，而[概念页面](/concepts/hot-module-replacement)提供了更多关于它的工作原理以及为什么它有用的细节。
+模块热替换(Hot Module Replacement 或 HMR)是 webpack 提供的最有用的功能之一。它允许在运行时更新各种模块，而无需进行完全刷新。本页面重点介绍 __实现__，而[概念页面](/concepts/hot-module-replacement)提供了更多关于它的工作原理以及为什么它有用的细节。
```
Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: 这里不需要空格

File: content/guides/hot-module-replacement.md [hunk 5]
```diff
@@ -61,11 +62,13 @@ __webpack.config.js__
   };
 ```
 
-You can also use the CLI to modify the [webpack-dev-server](https://github.com/webpack/webpack-dev-server) configuration with the following command: `webpack-dev-server --hotOnly`.
+你也可以通过命令来修改 [webpack-dev-server](https://github.com/webpack/webpack-dev-server) 的配置：
```
Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: 行数要完全对应起来。下面的 `webpack-dev-server --hotOnly` 应当在此行

File: content/guides/hot-module-replacement.md [hunk 6]
```diff
@@ -145,42 +147,41 @@ __index.js__
...
 
-First let's install both loaders with the following command:
+所以，可以使用以下命令安装两个 loader ：
 
 ```bash
 npm install --save-dev style-loader css-loader
 ```
 
-Now let's update the configuration file to make use of the loader.
+接下来我们来更新 webpack 的配置，让这两个 loader 生效：
```
Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: 末尾是句号

File: content/guides/hot-module-replacement.md [hunk 7]
```diff
@@ -289,7 +290,6 @@ __styles.css__
   }
 ```
```
Comment (dear-lizhihua) [inline content/guides/hot-module-replacement.md]: 这里少了一行

**Pull request merged**